src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/PortableInfoboxPageObject.java

```diff
@@ -112,32 +113,28 @@ public String getCategoryLinkName() {
     return categoryLinkInInfobox.getText();
   }
 
-  public String getUrlFromInternalLinkaAfterPageIsLoaded() {
-    wait.forElementVisible(headerTitle);
-    return driver.getCurrentUrl();
-  }
-
   public String getDataImageName() {
-    wait.forElementVisible(pInfoImageTag);
-    return pInfoImageTag.getAttribute("data-image-name");
+    wait.forElementVisible(imageTag);
+    return imageTag.getAttribute("data-image-name");
   }
 
-  public PortableInfoboxPageObject clickExternalLink() {
-    wait.forElementVisible(pInfoExternalLink);
-    pInfoExternalLink.click();
+  public PortableInfoboxPageObject clickExternalLinkWithIndex(int index) {
+    wait.forElementVisible(externalLink.get(index));
+    scrollToElement(externalLink.get(index));
+    externalLink.get(index).click();
     return this;
   }
 
-  public PortableInfoboxPageObject clickInternalLink(int index) {
-    wait.forElementVisible(pInfoInternalLinkList.get(index));
-    scrollToElement(pInfoInternalLinkList.get(index));
-    pInfoInternalLinkList.get(index).click();
+  public PortableInfoboxPageObject clickInternalLinkWithIndex(int index) {
```

Comment:
Looks like approach proposed in line 92 would help here too. The less code duplication, the better :)